static/js/components/ObjPageAnnotations.jsx

```diff
@@ -112,6 +122,38 @@ const ObjPageAnnotations = ({ annotations }) => {
     },
   ];
 
+  // if there are any spectra given, we should add the spectrum annotations after object annotations
+  if (spectra.length > 0) {
+    // first add the observed_at property to each annotation in tableData:
+    tableData.forEach((ann) => {
```

Comment:
Let's do this above where the regular obj annotations are added with a conditional check.

Reply:
Not sure what you mean?

Reply:
Above, there's an iteration over each annotation, where we should add this field if relevant there so we're not iterating again unnecessarily